cmd/ocitools/validate.go

```diff
@@ -61,6 +62,8 @@ var (
 		"CAP_KILL",
 		"CAP_AUDIT_WRITE",
 	}
+
+	configSchemaTemplate = "https://raw.githubusercontent.com/opencontainers/runtime-spec/v%s/schema/schema.json"
```

Comment:
If we are compiling ocitools for a particular runtime-spec version, it makes sense to use the version from the same schema by default.